master/SendImages.py

```diff
@@ -0,0 +1,39 @@
+import requests
+import argparse
+import random
+
+from DataBase.Redis import Redis
+
+class SendImages:
+    def __init__(self) -> None:
+        self.args = self.get_arguments()
+
+    def post(self, url: str, data: str) -> requests.Response:
+        return requests.post(url, json=data)
+
+    def find_worker(self) -> str:
+        redis = Redis()
+        print(redis.read(random.choice(redis.read_all())))
+        return f"http://{redis.read(random.choice(redis.read_all()))['ip']}:18081"
+
+    def send_image(self, worker_url: str) -> None:
+        data = self.args.image_name
+        result = self.post(f"{worker_url}/pull_image", data)
+
+        if result.status_code == 200:
+            print("Successfully pulled image")
+        else:
+            print(result.text)
+            raise Exception(f"Failed to pull image: {data}. {result.status_code}")
+
+    def get_arguments(self) -> argparse.Namespace:
```

Comment:
It's not responsibility of `ImageSender` class to get parameters via CLI (Single Responsibility principle).
Please pass all of the variables that the class needs as parameters.